FEXCore/Source/Interface/IR/IR.h

```diff
@@ -318,6 +358,7 @@ class OrderedNode final {
   }
 
   void AddUse() {
+    LOGMAN_THROW_A_FMT(NumUses < 65535, "never exceeded in practice");
```

Comment:
I wonder if this would get hit with older XeSS and large enough blocks.

Reply:
I have a long-standing yak shave to remove NumUses, perhaps I should just do that as a prerequisite for this here yak shave ....

Reply:
I've dropped this change. OrderedNode is grown a bit which is unfortunate, but I measured only a slight % difference attributable here, and this series is still a win overall to JIT time. So we should be able to merge it now and then circle back to this later for even more zoomies.